models/User.ts

```diff
@@ -0,0 +1,47 @@
+import type { roles, province, locale } from "@prisma/client";
+
+/* Input type for Parent inputs */
+export type ParentInput = {
+    id: string;
+    phoneNumber: string;
+    isLowIncome?: boolean;
+    addressLine1: string;
+    addressLine2?: string;
+    postalCode: string;
+    cityName: string;
+    province: province;
+    preferredLanguage: locale;
+};
+
+/* Input type for Volunteer inputs */
+export type VolunteerInput = {
+    id: string;
+    phoneNumber?: string;
+    isValid?: boolean;
+    backgroundFormLink?: string;
```

Comment:
Curious question, will they have to upload the form at the same time they create their account? Or do they enter the other info first, create the account, and upload the form later? If they upload it later, do we need this background form link field in the input 
Or do we leave the field here but enter null (since it has a ?)😅

Reply:
Good question, I'd assume they create the account first and upload the form later, so yeah I suppose you'd leave this field `null` until then - @feng-eric any comment?

Reply:
Yeah, I agree. It makes more sense to upload the form later.

Reply:
After talking to designers, they said it should be optional rather than setting it as null

Reply:
Ok thanks for the clarification!